### ISSUE TITLE
Move legal landing page text before new landing page is implemented

### ISSUE THREAD

**Issue by llienfec** [18F/fec-cms#1393]
So that we're ready to go live with legal landing pages, content team needs to review the current landing page.  We will make sure that all of the text is included in topic-specific landing pages (like Enforcement, or Regulations) so that we're not losing information when we move to the new format.

### To do:
- [x] Review design issue that shows where each of the links on the new landing page will go.
- [x] Move necessary text over and make sure the topic-specific pages are clear enough as stand-alone pages
- [x] Have @rhoughfec and @dorothyyeager review
- [x] Update pages and have them ready to go before new legal landing page goes live

cc: @rhoughfec @dorothyyeager

**Comment by llienfec** [comment]
Locked out of cloud.gov for now, so recording the text edits here and will make the changes shortly. Links listed out by Jen in https://github.com/18F/openFEC-web-app/issues/2288 were really helpful!

**AOs** - add after first paragraph:
> This archive contains advisory opinions from 1975 to the present. It also contains documents related to advisory opinions — such as requests, drafts and public comments — from 1990 to the present.

**Enforcement** - change header to the following.  Order of last paragraph was changed to match order on the page (even though Audits come before MURs).  Added complaints to list.  Note for future work: Enforcement page could do with some rewording/reorganizing.
>The FEC has exclusive jurisdiction over the civil enforcement of the federal campaign finance law.

> FEC enforcement matters, called Matters Under Review (MURs), are primarily handled by the Office of General Counsel. Enforcement cases must be kept confidential until the cases are closed.

> Other programs designed to augment the Office of General Counsel’s enforcement role include the Audit Program, complaints process, Alternative Dispute Resolution Program, and Administrative Fine Program.

**Regulations** - no changes.  (Too specific and we'll forget to update it every year; redundant w/ page info)

**Statutes** - add this after first paragraph:
> This archive contains the portions of Title 52 and Title 26 that are administered by FEC.

**Legislation** - add "to Congress" at the end of first paragraph.

**Court cases** - no changes, current page language is awesome.

**Audit** - no changes; everything appears to be covered but with different wording.

**Policy, etc.** - currently goes to transition and so I can't make changes.  For future information, the page currently says:
> Access statements, interpretive rules and other guidance issued by the FEC.

**Comment by llienfec** [comment]
Edits and blockers:
- [ ] **Advisory opinions** - No changes made b/c page is hard coded.  (blocked)
- [ ] **Enforcement** - kept language the same from old page.  A more explanatory header didn't make sense given the content and its order on the rest of the page.  Entire page needs a re-think after the landing pages change.
- [ ] **Statutes** -  No changes made b/c page is hard coded.  (blocked)
- [ ] **Legislation** - change made
- [ ] **Policy, etc.** - goes to transition

Since none of the of the information that would be added is vital to understanding the content of these pages, I suggest closing this issue for now, and we can pick it back up when we have more resources.  All of the wording changes are captured in the comment above.

**Comment by jenniferthibault** [comment]
@llienfec I can help change any language you need in the hard coded pages. Let me know!

**Comment by llienfec** [comment]
Thanks @jenniferthibault!  If you can add the AO page language that explains what the archives contain, that would be great, and I think it's all we need.  

The page already has the paragraph: Advisory opinions are official Commission responses to questions about how federal campaign finance law applies to specific, factual situations.

Add this as a second paragraph:   This archive contains advisory opinions from 1975 to the present. It also contains documents related to advisory opinions — such as requests, drafts and public comments — from 1990 to the present.

**Comment by AmyKort** [comment]
@llienfec  Are these changes on this page https://www.fec.gov/data/legal/advisory-opinions/ ? If so, could we also tackle https://github.com/18F/fec-cms/issues/1435 which would make "requesters" read "requestors" (note the o) in the feature flag? It seems like an easy fix if we're already editing that page. cc @jenniferthibault

**Comment by llienfec** [comment]
@AmyKort Yep, that's the page!

**Comment by jenniferthibault** [comment]
On it!

**Comment by jenniferthibault** [comment]
@llienfec the AO text was merged in as part of #1442. Would you like to see the Statutes text moved before closing this issue or is that no longer moving?

**Comment by llienfec** [comment]
Thanks @jenniferthibault.  Let's wait on the rest.  All are incredibly minor language changes that we can do in the future if needed.  Thank you so much for doing this!
